patches/services/fabricv4/20240227-add-package-alias-to-custom-path-param-type.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/services/fabricv4/test/api_cloud_routers_test.go b/services/fabricv4/test/api_cloud_routers_test.go
+index 4221152..eaf2468 100644
+--- a/services/fabricv4/test/api_cloud_routers_test.go
++++ b/services/fabricv4/test/api_cloud_routers_test.go
+@@ -82,7 +82,7 @@ func Test_fabricv4_CloudRoutersApiService(t *testing.T) {
+ 	t.Run("Test CloudRoutersApiService GetCloudRouterPackageByCode", func(t *testing.T) {
+ 		t.Skip("skip test") // remove to run test
+ 
+-		var routerPackageCode RouterPackageCode
++		var routerPackageCode openapiclient.RouterPackageCode
```

Comment:
This seems like something the generator should have figured out for us.  Worth digging into later when there's time.